test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -9600,6 +9713,51 @@ def import_hook(self, name, body, level=logging.INFO):
                         body + '---')
         return True
 
+    def import_hook_config(self, name, body, level=logging.INFO):
+        """
+        Helper function to import hook config by name.
+        The hook must have been created prior to calling this
+        function.
+
+        :param name: The name of the hook to import body to
+        :type name: str
+        :param body: The body of the hook config as a string.
+        :type body: str
+        :returns: True on success.
+        :raises: PbsManagerError
+        """
+        # sync_mom_hookfiles_timeout is 15min by default
+        # Setting it to lower value to avoid the race condition at hook copy
+        srv_stat = self.status(SERVER, 'sync_mom_hookfiles_timeout')
```

Comment:
where do you use srv_stat?

Reply:
Function is no longer required since i have moved to qmgr command for getting hooks output and importing it